app/metrics.yaml

```diff
@@ -83,3 +83,168 @@ searches:
       - fxr-telemetry@mozilla.com
       - dmu@mozilla.com
     expires: "2020-05-01"
+
+tabs:
+  opened:
+    type: labeled_counter
+    description: >
+      Number of opened tabs
+    send_in_pings:
+      - session_end
+    labels:
+      - context_menu
+      - tabs_dialog
+      - bookmarks
+      - history
+      - fxa_login
+      - received
+      - pre_existing
+      - browser
+    bugs:
+      - https://github.com/MozillaReality/FirefoxReality/issues/1609
+    data_reviews:
+      - https://github.com/MozillaReality/FirefoxReality/pull/2327#issuecomment-558139174
+    notification_emails:
+      - fxr-telemetry@mozilla.com
+      - manmartin@mozilla.com
+    expires: "2020-05-01"
+
+  activated:
+    type: counter
+    description: >
+      Number of activated tabs
+    send_in_pings:
+      - session_end
+    bugs:
+      - https://github.com/MozillaReality/FirefoxReality/issues/1609
+    data_reviews:
+      - https://github.com/MozillaReality/FirefoxReality/pull/2327#issuecomment-558139174
+    notification_emails:
+      - fxr-telemetry@mozilla.com
+      - manmartin@mozilla.com
+    expires: "2020-05-01"
+
+firefox_account:
+  sign_in:
+    type: event
+    description: >
+      A user pressed the sign in button on the sync account page
```

Comment:
To better understand your use-case: what do you need this telemetry for? Do you need this only for counting how many times does a sign-in happen?

If what you really care about is a count, then you should probably use a `counter` metric type.

Reply:
We try to answer the following questions:
- number of users who click the "sign in" button
- number of users who successfully log-in to an account
- number of users who sign out

So we don't only need a raw count of how many users login successfully but we also want to know how many users drop because FxA login issues.

Reply:
@keianhzo the shape of the data really depends on _how_ are you planning on answering the question :) For example, "how many users drop because FxA login issues". How are you planning on answering this question using events?

Reply:
Humm, I don't really know about how to use this metrics on the backend, I think I'll need @daoshengmu's help to answer this better understand that.

Reply:
- number of users who click the "sign in" button
If we wanna analysis how many **USERS** by daily/weekly for tracking ppl who clicking sign-in, this looks like a metric for UI interaction. I would choose using `event` instead of `counter` and `lifetime:ping` then doing count_distinct(client_id) at queries.

- number of users who successfully log-in to an account
It looks like we wanna know how many **USERS** sign-in success or fail by daily? For this case, I would expect to have a sign_in_result `event` metric, then giving extra {"state": true or false } instead of using two different `event` metrics. Then, doing count_distinct(client_id) at queries to check if this event state is true or false.

- number of users who sign out
Same as "sign in" button

Reply:
Changed